pom.xml

```diff
@@ -326,43 +326,43 @@
             </configuration>
             <executions>
               <execution>
-                <id>default-prepare-agent</id>
+                <id>prepare-jacoco-agent</id>
                 <goals>
                   <goal>prepare-agent</goal>
                 </goals>
                 <configuration>
-                  <destFile>${basedir}/target/coverage-reports/jacoco-unit.exec</destFile>
+                  <destFile>${project.build.directory}/coverage-reports/jacoco-unit.exec</destFile>
                 </configuration>
               </execution>
               <execution>
-                <id>default-report</id>
+                <id>prepare-package-jacoco-report</id>
                 <phase>prepare-package</phase>
                 <goals>
                   <goal>report</goal>
                 </goals>
                 <configuration>
-                  <dataFile>${basedir}/target/coverage-reports/jacoco-unit.exec</dataFile>
+                  <dataFile>${project.build.directory}/coverage-reports/jacoco-unit.exec</dataFile>
                 </configuration>
               </execution>
               <execution>
-                <id>pre-integration-test</id>
+                <id>pre-integration-test-prepare-jacoco-agent</id>
```

Comment:
:disappointed:

Reply:
Sorry I tried to come up with a better name, but it's often difficult to think of something other than phase + goal.
In most code samples I've come across it's just default-<phase>